modules/ROOT/pages/getting-started/index.adoc

```diff
@@ -121,36 +129,33 @@ console.log(`🚀 Server ready at ${url}`);
 
 == Start the server
 
-Finally, you're ready to start up your GraphQL server! Back in the command line, run the following command:
+Finally, you are ready to start up your GraphQL server. 
+Back in the command line, run:
 
 [source, bash, indent=0]
 ----
 node index.js
 ----
 
-All going well, you should see the following output:
+If successful, you should see the following output:
 
 [source, bash, indent=0]
 ----
 🚀 Server ready at http://localhost:4000/
 ----
 
-Where http://localhost:4000/ is the default URL which Apollo Server starts at.
-
-== Create your first nodes in the database
-
-Now it's time to add some data to your Neo4j database using your GraphQL API!
-
-Visit http://localhost:4000/ in your web browser and you'll see the following landing page:
-
-image::apollo-server-landing-page.png[title="Apollo Server Landing Page"]
+The address http://localhost:4000/ is the default URL in which Apollo Server starts at.
 
-Click "Query your server" which will open the Sandbox.
+== Create nodes in the database
 
-image::first-mutation.png[title="First mutation"]
-
-At the moment your database is empty! To get some data in there, you can create a movie and an actor in that movie, all in one mutation. The mutation in the screenshot above can also be found below:
+. Visit http://localhost:4000/ in your web browser. 
+You should get redirected to the Sandbox:
```

Comment:
```suggestion
You should get redirected to the Apollo Sandbox:
```
I'm not 100% sure if it's required to mention it is the Apollo Sandbox we're referring to, but it was specified before.